shared/actions/platform-specific/index.native.tsx

```diff
@@ -646,6 +646,14 @@ export const watchPositionForMap = async (errFn: () => void): Promise<number> =>
   return watchID
 }
 
+const configureFileAttachmentDownloadForAndroid = () =>
+  RPCChatTypes.localConfigureFileAttachmentDownloadLocalRpcPromise({
+    // Android's cache dir is (when I tried) [app]/cache but Go side uses
+    // [app]/.cache by default, which can't be used for sharing to other apps.
+    cacheDirOverride: RNFetchBlob.fs.dirs.CacheDir,
```

Comment:
@MarcoPolo this ok

Reply:
Some more context: this is actually the same path that we've been using for Android download. I just plumbed this into Go side. Fs uses the same cache dir.